sdk/RELEASE.md

```diff
@@ -18,7 +18,7 @@
 * Support re-use of PVC with VolumeOp. [\#6582](https://github.com/kubeflow/pipelines/pull/6582)
 * When namespace file is missing, remove stack trace so it doesn't look like an error [\#6590](https://github.com/kubeflow/pipelines/pull/6590)
 * Local runner supports additional docker options. [\#6599](https://github.com/kubeflow/pipelines/pull/6599)
-
+* Fix the error that kfp v1 compiler failed to provide unique name for ops of the same component. [\#6600](https://github.com/kubeflow/pipelines/pull/6600)
```

Comment:
This actually affected both v1 and v2 compiler. 
@ji-yaqi can you please fix this description when you do the release? Thanks!